### DISCUSSION TITLE
Next release

### DISCUSSION THREAD

**Discussion by Robzak80** [koreader/koreader#12161] [General]
Hi
I have been waiting for the new release that supports my libra kobo colour. This will be the first time I install Koreader. When will the new release be out? 
Regards
Rob

**Comment by mergen3107** [comment]
Install latest nightly from http://build.koreader.rocks/download/nightly/

or use one-lick installer with two-days old nightly from https://www.mobileread.com/forums/showthread.php?t=314220

both are good for you

**Reply by Robzak80** [reply]
But nighty might have bugs and anyway I will have to reinstall Koreader when the stable release is out

**Reply by mergen3107** [reply]
:D nope! Stable can be as buggy as nightly. Or rather, nightly is now as stable as Stable.

**Reply by Frenzie** [reply]
> But nighty might have bugs

Depends when you install a nightly. In this case it'll be identical or very close to it.

**Comment by Robzak80** [comment]
Downloaded and installed. Working great so far. Thanks team. This is much better than the native kobo UI experience
